src/openzeppelin/introspection/interface.cairo

```diff
@@ -0,0 +1,11 @@
+const ISRC5_ID: felt252 = 0x3f918d17e5ee77373b56385708f855659a07f75997f365cf87748628532a055;
+
+#[starknet::interface]
+trait ISRC5<TState> {
```

Comment:
I had the same conflict, but I'm leaning towards using `TState`, because we will be using that a lot (in all the interfaces), and using `TContractState` would be an important boilerplate IMO. Also, the concrete type doesn't have to match the generic param. We usually use `<T>`, and we replace the actual type that can be different, like `Span<felt>` for `Span<T>` in corelib.

Reply:
Mmm right, when you make the point of just using `T`, I suppose even using `TState` is an added descriptor. Sounds good to me, `TState` it is!